src/peft/tuners/lora/layer.py

```diff
@@ -47,6 +48,9 @@ def __init__(self, base_layer: nn.Module, **kwargs) -> None:
         # Mark the weight as unmerged
         self._disable_adapters = False
         self.merged_adapters = []
+        self.use_dora: dict[str, bool] = {}
```

Comment:
(nit): `use_dora` where the variable is supposed to be a `dict` -- doesn't seem like a good name. Could we go for something else?

Reply:
I agree in principle, but I wanted to be consistent with the name of the other model params, which are also dicts to support multiple adapters. E.g. `lora_alpha`, `scaling`, `lora_A`, `lora_B` are all dicts. If I would rename this to something like `use_dora_mapping`, it would be inconsistent.